content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -10,33 +10,29 @@ weight: 10
 
 {{% capture overview %}}
 
-ReplicaSet is the next-generation Replication Controller. The only difference
-between a _ReplicaSet_ and a
-[_Replication Controller_](/docs/concepts/workloads/controllers/replicationcontroller/) right now is
-the selector support. ReplicaSet supports the new set-based selector requirements
-as described in the [labels user guide](/docs/concepts/overview/working-with-objects/labels/#label-selectors)
-whereas a Replication Controller only supports equality-based selector requirements.
+A ReplicaSet's purpose is to maintain a stable set of replica Pods running at any given time. As such, it is often
+used to guarantee the availability of a specified number of identical pods.
+
 
 {{% /capture %}}
 
 {{% capture body %}}
 
-## How to use a ReplicaSet
+## How a ReplicaSet Works
+
+A ReplicaSet is defined with fields including, a selector which specifies how to identify pods it can acquire, a number
+of replicas indicating how many pods it should be maintaining, and a pod template specifying the data of new pods
+it should create to meet the number of replicas criteria. A ReplicaSet then fulfills its purpose by creating
+and deleting Pods as needed to reach the desired number, using its given pod template for the creation.
```

Comment:
Ending with "for the creation" seems vague to me. Maybe this:
"...reach the desired number. When a ReplicaSet needs to create new Pods, it uses its Pod template."